subprojects/core/src/integTest/groovy/org/gradle/api/tasks/AbstractDirectorySensitivityIntegrationSpec.groovy

```diff
@@ -212,6 +212,8 @@ abstract class AbstractDirectorySensitivityIntegrationSpec extends AbstractInteg
     }
 
     def "artifact transforms are sensitive to empty directories by default"() {
+        requireOwnGradleUserHomeDir()
```

Comment:
🤔 I'd convert the transform into an incremental transform instead, so the test would exercise the build cache and up-to-date checks as the other tests in that class, though I don't feel strongly about it.